src/router.js

```diff
@@ -1,4 +1,16 @@
-//1. routes all requests from index.html;
-//2. routes search request --> will call request.js;
-//3. routes unknown url;
-//go to handlers;
+const path = require("path");
+const handlers = require("./handlers");
+
+const router(request, response) => {
+  let endpoint = request.url;
+  if (endpoint === '/') {
+    handlers.serveLanding(request, response)
+  } else if (endpoint.indexOf('public') !== -1) {
```

Comment:
I think we have now lost our whitelist functionality. Any requested url with 'public' in it would be sent to serveLanding which will concatenate the requested url into a filepath. This could be anything, such as 'public/../package.json'. The original intent was for the contentType object to function as a whitelist, so that only if it contained the requested resource as a key, would the file read be executed. Otherwise, 404 would be sent back.